src/components/Admin/AddUserForm.js

```diff
@@ -21,52 +25,85 @@ const switchLayout = {
     wrapperCol: { offset: 20 },
 };
 
-
 /*End Form Styling*/
+
+const initialValues = {
+    first_name: '',
+    last_name: '',
+    phone: ''
+};
+
 const AddUserForm = props => {
-    const { EditMode, Record } = props;
+    const { EditMode, Record, formUpdated } = props;
```

Comment:
I would move this deconstruction into the props placement so something like:
```javascript
const AddUserForm = { EditMode, Record, formUpdated} =>{}
```

Reply:
Also, props should start with a lower case letter. Upper case should be reserved for components. If you want to call something out as a constant use `ALL_CAPS`

Reply:
Thank you! I will work on getting these items corrected. I appreciate the suggestions.